doc-tool/src/dotty/tools/dottydoc/model/references.scala

```diff
@@ -67,6 +61,7 @@ object references {
         s"$title: $byName${ref.showReference}$repeated"
 
       case ConstantReference(title) => title
+      case EmptyReference => ""
```

Comment:
Does it ever happen?

Reply:
I did no happen so far. Will put an assert